packages/browser-sdk/src/solana/connect.ts

```diff
@@ -1,7 +1,9 @@
-import { getProvider } from "./getProvider";
+import { getProvider as defaultGetProvider } from "./getProvider";
+import type { SolanaOperationOptions } from "./types";
```

Comment:
I don't see this type in this PR

Reply:
to summarize:
is good to have access to a different provider, not only window.phantom
but there are better ways to inject the provider rather than passing it as a parameter in all the functions
i will change my PR to remove those options and find a way to define the provider in the createSolanaPlugin options
:+1:  :-1:  ?